crates/nu-command/src/dataframe/eager/open.rs

```diff
@@ -22,7 +22,7 @@ impl Command for OpenDataFrame {
     }
 
     fn usage(&self) -> &str {
-        "Opens csv, json, arrow, or parquet file to create dataframe."
+        "Opens CSV, json, arrow, or parquet file to create dataframe."
```

Comment:
If we do this, JSON is also all caps

Reply:
I agree, but I though about doing this in a seperate pull request.
What do you think?

Reply:
I think this is a can-of-worms type thing. You'll be searching the entire repo for csv, json, yaml, yml, eml, ics, nuon, ods, parquet, ssv, toml, tsv, vcf, xlsx, xml, etc. I'm personally fine with leaving them all lowercase.

Reply:
> I though about doing this in a separate pull request.

That's too granular IMO; if you have multiple small spelling fixes I would prefer 1 pull request.

Reply:
OK, I'll do further "acronym fixes" in this pull request. I mark it as a draft.

Reply:
The PR has now fixes for both CSV (ceef0ba067cad437de2a65e71b5ff5ba4649583a) and JSON (331feb2d9ac6bc4b83d217df6b40e3bd32259ea3) acronyms. 🤓